src/board-explorer-app/board-explorer-app.html

```diff
@@ -470,12 +470,12 @@
             </div>
             <div hidden$="[[isEmpty(active.connected)]]">
               Connected to: <template is="dom-repeat" items="[[active.connected]]" as="refdes">
-                <a href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
+                <a on-mouseover="onAnchorHoverIn" on-mouseout="onAnchorHoverOut" href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
               </template>
             </div>
             <div hidden$="[[isEmpty(active.related)]]">
               Related to: <template is="dom-repeat" items="[[active.related]]" as="refdes">
-                <a href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
+                <a on-mouseover="onAnchorHoverIn" on-mouseout="onAnchorHoverOut" href="#[[board]]/[[refdes]]">[[lookupName(refdes)]]</a>
               </template>
```

Comment:
Determine if we add the listeners based on if the href is a hash URL with a reference after it.